ReceiveMessage/models/UserState.ts

```diff
@@ -1,11 +1,22 @@
-import mongoose, { Schema } from 'mongoose';
+import mongoose, { Schema, Document } from 'mongoose';
+import ChatbotMessage;
```

Comment:
import ChatbotMessage from './ChatbotMessage', but we don't actually need the schema itself, we only need the interface for the schema, see below comment. Import interfaces/types around curly braces, like
`import {myTyping} from 'path'`